.doc_gen/metadata/redshift_metadata.yaml

```diff
@@ -240,6 +280,14 @@ redshift_Query:
     redshift: {Query}
 redshift_DescribeStatement:
   languages:
+    Go:
+      versions:
+        - sdk_version: 2
+          github: gov2/redshift
+          excerpts:
+            - description:
+              snippet_tags:
+                - gov2.redshift.DescribeStatement
```

Comment:
This is a redshiftdata client action so you can't include it as part of redshift.
The API link is broken (this is broken for all languages, BTW).

Reply:
Removed